news/_posts/2020-03-11-improvements-on-contributors-board.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "Improvements to come on contributors board"
+subtitle: "More features, more modern, and more information about contributors"
+date:  2020-03-12 15:30:00
+authors: [AntoineThomas]
+icon: icon-board
+image: /assets/images/theme/meta-logo-build.png
+tags: [contribute, community]
+---
+
+
+After all those years without any modification nor improvements, it is really time to think about the future of the "[Top contribution achievements](http://contributors.prestashop.com/)" board. While discussing with many people from the community on how to open the governance of the open source project, and how to involve more contributors, new needs have been identified. This article is about quick wins, upcoming developments, and about the general spirit behind this work.
+
+## But, why?
+
+In 2020, a lot of work is going to be done around the PrestaShop open source project to make it more attractive for contributors. It must of course take in account that nobody really contributes voluntarily, because at the end of the month, everybody has to pay their bills. So, like with other famous open source projects, people and companies who takes time to contribute must be able to show it: being a visible expert in the community has a lot of value to attract customers, it proves that they know what they are doing with their PrestaShop business.
+
+There are many ideas that are currently being tested, or will be soon:
+
+- [Call for papers at PrestaShop events](https://build.prestashop.com/news/CFP-PrestaShop-Day-Paris-2020/), giving a place for community speakers
+- Opening the maintainer role on a GitHub repository with selected developers, in order to idenfiy problems and document how to do it well
+- Opening a chat platform for contributors (due date on Q1)
```

Comment:
due date before the end of month ?

Reply:
```suggestion
- Opening a chat platform for contributors (due date on Q1);
```

Reply:
yes, I know, this is short...

Reply:
```suggestion
- Opening a chat platform for contributors (it's planned to happen very soon);
```

Reply:
should we keep the ";"?
if yes, i should add them everywhere on lists.